packages/utxo-lib/tests/__fixtures__/compose.crosscheck.ts

```diff
@@ -1,6 +1,23 @@
 import { UTXO } from './compose';
+import {
+    ComposeChangeAddress,
+    ComposeInput,
+    ComposeOutput,
+    ComposeRequest,
+} from '../../src/types/compose';
+import { CoinSelectPaymentType } from '../../src';
 
-export const fixturesCrossCheck = [
+type AnyComposeRequest = ComposeRequest<ComposeInput, ComposeOutput, ComposeChangeAddress>;
+
+type Fixture = {
+    description: string;
+    request: Omit<AnyComposeRequest, 'network'> & {
+        network?: AnyComposeRequest['network'];
+    };
```

Comment:
IMHO we should give some thought to explore https://www.npmjs.com/package/@total-typescript/shoehorn 

Not the scope of this PR ofc, this is ok :slightly_smiling_face: 

Also, don't we have some type generic util for Nullable fields on object? Can't find it, so maybe I'm wrong.